app/javascript/controllers/index.js

```
@@ -10,10 +10,14 @@ import ProjectsController from './projects_controller';
 import NotificationsController from './notifications_controller';
 import ContestController from './contest_controller';
 import SearchBarController from './search_bar_controller';
+import ShareController from './share_controller';
```

ESLint

[error] Parse errors in imported module './share_controller': Unexpected token = (4:17)

(import/no-named-as-default)